doc/how-to-write-conformance-tests.md

```diff
@@ -128,21 +171,97 @@ pub mod my_module {
 **Instantiated Tests in `my_impl`:**
 
 ```rs
-use iceoryx2_bb_testing::instantiate_conformance_tests;
+use iceoryx2_bb_testing::instantiate_conformance_tests_with_module;
+use my_impl::{ImplA, ImplB};
+
+instantiate_conformance_tests_with_module!(
+    impl_a,
+    my_test_crate::my_module,
+    super::ImplA
+);
+
+instantiate_conformance_tests_with_module!(
+    impl_b,
+    my_test_crate::my_module,
+    super::ImplB
+);
+```
+
+This will generate and run `test_feature_x` and `test_feature_y` for both
```

Comment:
Language here (and in the following) is inconsistent with the introduction, which refers to the (undefined) "System Under Test". I personally like how it is described here better, the "System Under Test" is a little ambiguous to me.

Reply:
My proposal is already added to `## 1. Overview`, is this sufficient from your point of view?